test/bulk_insert/worker_test.rb

```diff
@@ -159,11 +160,13 @@ class BulkInsertWorkerTest < ActiveSupport::TestCase
     worker.add greeting: "second"
     worker.save!
     assert_equal 1, worker.result_sets.count
```

Comment:
my guess here is that the result_sets for non postgresql adapters would look like `[ nil ]`